templates/taxi/car_detail.html

```diff
@@ -3,20 +3,37 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
-  <hr>
+
+    {% if car in user.cars.all %}
+      <a href="{% url 'taxi:add_delete_driver' car.id %}" class="btn btn-danger link-to-page">
+        Delete me from this car
+
+      </a>
+    {% else %}
+      <a href="{% url 'taxi:add_delete_driver' car.id %}" class="btn btn-success link-to-page">
+        Assign me to this car
+      </a>
+    {% endif %}
+
   <ul>
```

Comment:
if you have ul tag, you need to have li as well. Update/Delete buttons should not be inside ul